lib/launches/view/launches_page.dart

```diff
@@ -0,0 +1,181 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:intl/intl.dart';
+import 'package:launch_repository/launch_repository.dart';
+import 'package:spacex_demo/l10n/l10n.dart';
+import 'package:spacex_demo/launches/cubit/launches_cubit.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class LaunchesPage extends StatelessWidget {
+  const LaunchesPage({Key? key}) : super(key: key);
+
+  static Route<LaunchesPage> route() {
+    return MaterialPageRoute(
+      builder: (context) => const LaunchesPage(),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (_) => LaunchesCubit(
+        launchRepository: context.read<LaunchRepository>(),
+      )..fetchLatestLaunch(),
+      child: const LaunchesView(),
+    );
+  }
+}
+
+class LaunchesView extends StatelessWidget {
+  const LaunchesView({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final l10n = context.l10n;
+
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(l10n.latestLaunchAppBarTitle),
+      ),
+      body: const Center(
+        child: _Content(),
+      ),
+    );
+  }
+}
+
+class _Content extends StatelessWidget {
+  const _Content({Key? key}) : super(key: key);
```

Comment:
```suggestion
      body: const Center(
        child: _LaunchesContent(),
      ),
    );
  }
}

class _LaunchesContent extends StatelessWidget {
  const _LaunchesContent({Key? key}) : super(key: key);
```

minor but thoughts on adjusting the name?